src/__tests__/runTest.test.ts

```diff
@@ -14,7 +14,7 @@ import runTest from '../run';
 
 jest.mock('tsd-lite');
 jest.mock('../formatter', () => ({
-  ...jest.requireActual('../formatter'),
+  ...(jest.requireActual('../formatter') as typeof import('../formatter')),
```

Comment:
No generic?

Reply:
Right. Only that Jest is not yet published with generic support. Please don’t hurry, we have to revert `toBeCalled` types. Unfortunately they are buggy. I will do this later this week.